File: Implement Queue using Stacks.js
class MyQueue {
    data = [];
    constructor() {
        this.data = [];
    }
/** 
 * @param {number} x
 * @return {void}
 */
    push(x) {
        this.data.push(x);
    }

/**
 * @return {number}
 */
    pop() {
     
        const temp=this.data.shift()
 
       return temp
    }

/**
 * @return {number}
 */
    peek() {
        return this.data[0];
    }


/**
 * @return {boolean}
 */
    empty() {
        return this.data.length === 0
    }
}
/** 
 * Your MyQueue object will be instantiated and called as such:
 * var obj = new MyQueue()
 * obj.push(x)
 * var param_2 = obj.pop()
 * var param_3 = obj.peek()
 * var param_4 = obj.empty()
 */

File: Implement Queue using Stacks.java
class MyQueue {

    public MyQueue() {
        
    }
    
    Stack<Integer> s=new Stack<>();
      
    public void push(int x) {
           s.push(x);
    }
    
    public int pop() {
           int rem_ele = s.remove(0);
           return rem_ele;
    }
    
    public int peek() {
           return s.firstElement();
    }
    
    public boolean empty() {
           return s.empty();
    }
}

/**
 * Your MyQueue object will be instantiated and called as such:
 * MyQueue obj = new MyQueue();
 * obj.push(x);
 * int param_2 = obj.pop();
 * int param_3 = obj.peek();
 * boolean param_4 = obj.empty();
 */
